volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -1262,16 +1264,41 @@ def get_mac_address(self):
 
         return self._format_as_mac_address(hwaddr)
 
+    def _get_flag_choices(self) -> Dict:
+        """Return the net_deivce flags as a list of strings"""
+        vmlinux = linux.LinuxUtilities.get_module_from_volobj_type(self._context, self)
+        try:
+            # kernels >= 3.15
+            net_device_flags_enum = vmlinux.get_enumeration("net_device_flags")
+            choices = net_device_flags_enum.choices
+        except exceptions.SymbolError:
+            # kernels < 3.15
+            choices = NET_DEVICE_FLAGS
+
+        return choices
+
+    def _get_net_device_flag_value(self, name):
+        """Return the net_deivce flag value based on the flag name"""
+        return self._get_flag_choices()[name]
```

Comment:
I'd be happier if this were a get with a default value?  Having said that, it's such a short function, perhaps it's not worth having at all (given it's private so can't be used by anyone else?)

Reply:
Default value and error managed. 
I prefer to keep it as a function, even though it's short and private. This way, it serves the purpose of documenting what it does and increases the code's readability. I would let the language to implement this inline if needed.
Anyway, if you still think it should be inline, let me know, and I will change it.